calico-cloud_versioned_docs/version-3.16/image-assurance/scanners/cluster-scanner.mdx

```diff
@@ -1,62 +1,97 @@
 ---
-description: Identify vulnerabilities in Kubernetes clusters.
+description: Detect vulnerabilities in a Kubernetes cluster.
 ---
 
-# Enable automatic image scanning inside a cluster
+# Scan all images in a Kubernetes cluster
 
-## Big picture
+Scan all images in a Kubernetes cluster for continuous clean bill of health and defense in depth.
```

Comment:
Not sure about the "continuous clean bill of health and defense in depth" part here. 

The cluster scanner certainly does run continuously, but what does "clean bill of health and defense in depth" mean in this context?